File: Code3/Sed_sequences.py
T=int(input())

for i in range(T):
    N,K=map(int,input().strip().split(" "))
    
    A=list(map(int,input().strip().split(" ")))[:N]
    sum=0
    for j in A:
        sum=sum+j
    
    if sum%K==0:
        print("0")
    else:
        print("1")

File: Code3/Sed_Sequences.c
#include <stdio.h>

int main(void) {
	int T,N,K;
	
	scanf("%d",&T);
	
	int sum;
	
	for(int i=1;i<=T;i++)
	{
	    sum=0;
	    scanf("%d%d",&N,&K);
	    
	    int A[N];
	    for(int i=0;i<N;i++)
	    {
	        scanf("%d",&A[i]);
	        sum=sum+A[i];
	    }
	    if(sum%K==0)
	    {
	        printf("0\n");
	    }
	    else
	    {
	        printf("1\n");
	    }
	}
	
	return 0;
}

